packages/client/codegen.yml

```diff
@@ -11,7 +11,7 @@ overwrite: true
 schema:
   - http://localhost:7070/graphql:
       headers:
-        Authorization: 'Bearer ${TOKEN}'
```

Comment:
This we probably don't want to get in develop, right?

Reply:
Yes. I forget to change it back. Fixed it now.

Reply:
Sorry, did you push? I'm not seeing the change yet